src/catalogue/catalogue.component.tsx

```diff
@@ -230,16 +230,25 @@ function Catalogue() {
   const [selectedCatalogueCategory, setSelectedCatalogueCategory] =
     React.useState<CatalogueCategory | undefined>(undefined);
 
+  // useEffect hook to handle updates to selectedCatalogueCategory when catalogueCategoryData changes
```

Comment:
Could this be more specific on what it fixes, I have already forgotten 😄 - wasn't it to ensure the edit/add property dialogue would apply to the correct category?